test/index.js

```diff
@@ -124,8 +124,21 @@ function runTests (name, compat) {
   var algos = ['sha1', 'sha224', 'sha256', 'sha384', 'sha512', 'ripemd160']
   algos.forEach(function (algorithm) {
     fixtures.valid.forEach(function (f) {
-      var key = f.key || Buffer.from(f.keyHex, 'hex')
-      var salt = f.salt || Buffer.from(f.saltHex, 'hex')
+      var key, salt
+      if (f.keyUint8Array) {
+        key = new Uint8Array(f.keyuint8)
+      } else if (f.keyHex) {
+        key = Buffer.from(f.keyHex, 'hex')
+      } else {
+        key = f.key
+      }
+      if (f.saltUint8Array) {
+        salt = new Uint8Array(f.saltuint8)
```

Comment:
This property name is outdated. It should case a test fail, right? The same for the key case above

Reply:
Whoops, I missed that, it is fixed now. The tests for `ripemd160` are still failing for the Uint8Array, could you look into that? I am not very familiar with the hash libraries.

Reply:
> The tests for `ripemd160` are still failing for the Uint8Array, could you look into that? I am not very familiar with the hash libraries.

You added ripemd160 in the meantime, which seems to add an additional expected field to the tests that your orginal PR code does not have: https://github.com/crypto-browserify/pbkdf2/commit/9da9d9c4f426237e72b8f189ae14e3e473c84c69 Does this explain the issue?

Reply:
Oh, right, I rebased against the current master, hence the new test cases. Thank you, all tests are passing now.